assets/js/theme/global/compare-products.js

```diff
@@ -1,71 +1,73 @@
-import trigger from '../common/utils/trigger'
-import { showAlertModal } from './modal'
 import q$, { q$$ } from './selector'
 
-function decrementCounter(counter, item) {
-    const index = counter.indexOf(item)
-
-    if (index > -1) {
-        counter.splice(index, 1)
-    }
+const selectors = {
+    LSProducts: 'compare-products',
```

Comment:
LSProducts by reading the code below stands for local storage products. Can we rename it to `localStorageProducts` so that it is easier to understand what it stands for?